microsoft-edge/devtools-guide-chromium/whats-new/2022/05/devtools-102.md

```diff
@@ -0,0 +1,202 @@
+
+In the following image, in the **Filter edges** dropdown menu, **Hide cycled** is selected, so the cycled node is not displayed:
+
+![In the 'Filter edges' dropdown menu, 'Hide cycled' is selected.](devtools-102-images/filters-retainers-memory-tool-hide-cycled.png)
+
+_Cycles_ are nodes that appear at least twice in a retainer path.
+One appearance of a node is earlier in the retainer path, and other appearances of that node are later in the retainer path.
+To free up memory, it's most important to remove the occurrence of the node which appears first in the retainer path.
+The second and potentially subsequent appearances of the node are still displayed in the **Retainers** section.
+
+_Internal nodes_ are objects that are specific to V8 (the JavaScript engine in Microsoft Edge).  Internal nodes are also displayed in the **Retainers** section.
+
+See also:
+* [Cycles](../../../memory-problems/memory-101.md#cycles) in _Memory terminology_.
+
+
+<!-- ====================================================================== -->
+## The Inspect element tooltip now persists until you dismiss it
+
+<!-- Title: Improving the Inspect element tooltip -->
+<!-- Subtitle: The Inspect element tooltip now persists until you dismiss it, improving accessibility and ease of use. -->
+
+Previously, in the **Elements** tool, the overlay for the node that you selected only displayed on hover when you used the **Inspect tool** (![Inspect tool icon.](../../../media/inspect-tool-icon-light-theme.png)).  The previous design was difficult to use with a screen magnifier or other assistive technologies.
+
+In Microsoft Edge 102, when using the **Inspect** tool and moving around the rendered webpage, you can keep the current **Inspect** overlay displayed.  Press and hold `Ctrl`+`Alt` (Windows, Linux) or `Ctrl`+`Option` (macOS) while you move around in the rendered webpage.  The existing tooltip and grid color overlay for the **Inspect** tool remains displayed while you hover over different parts of the rendered webpage:
+
+![Inspect element tooltip now persists.](devtools-102-images/inspect-element-tooltip-persists.png)
+
+Also, you can now press `Ctrl` to hide the Inspect tool's overlay while you move the mouse pointer over the rendered webpage.
+
+For the history of this feature in the Chromium open-source project, see Issue [1316208](https://crbug.com/1316208).
+
+See also:
+*  [Persisting the Inspect tool's tooltip and grid color overlay](../../../css/inspect.md#persisting-the-inspect-tools-tooltip-and-grid-color-overlay) in _Analyze pages using the Inspect tool_.
+*  [Inspect an element on the page](../../../accessibility/navigation.md#inspect-an-element-on-the-page) in _Navigate DevTools with assistive technology_.
+
+
+<!-- ====================================================================== -->
+## Improving accessibility in high contrast mode
```

Comment:
```suggestion
## Controls are fully rendered in high-contrast mode
```
@zoherghadyali  & Olivia F - make heading more specific?  In what specific way is accessibility improved in high-contrast mode?

Reply:
It's hard to say whether the controls are fully bug-free across the DevTools because the UI is so large, and regressions are especially common in HC mode.  Maybe "Improved rendering of controls in high-contrast mode"?